rootfs/docker-entrypoint-initdb.d/001_setup_envdir.sh

```diff
@@ -40,4 +40,23 @@ elif [ "$DATABASE_STORAGE" == "azure" ]; then
   echo $WABS_ACCESS_KEY > WABS_ACCESS_KEY
   echo "wabs://$BUCKET_NAME" > WALE_WABS_PREFIX
   echo $BUCKET_NAME > BUCKET_NAME
+elif [ "$DATABASE_STORAGE" == "swift" ]; then
+  SWIFT_USER=$(cat /var/run/secrets/deis/objectstore/creds/username)
+  SWIFT_PASSWORD=$(cat /var/run/secrets/deis/objectstore/creds/password)
+  SWIFT_TENANT=$(cat /var/run/secrets/deis/objectstore/creds/tenant)
+  SWIFT_AUTHURL=$(cat /var/run/secrets/deis/objectstore/creds/authurl)
+  if [[ -e /var/run/secrets/deis/objectstore/creds/auth_version ]]; then
+    SWIFT_AUTH_VERSION=$(cat /var/run/secrets/deis/objectstore/creds/auth_version)
```

Comment:
can you just read `/var/run/secrets/deis/objectstore/creds/authversion` instead of `/var/run/secrets/deis/objectstore/creds/auth_version`